dev/alive/docker-compose.yaml

```
@@ -0,0 +1,38 @@
+version: "3"
+services:
+  alive-mssql:
+    extends:
+      file: ../common-services.yaml
+      service: mssql
+    volumes:
+      - leaf-alive-mssql:/var/opt/mssql
+
+  alive-coreapi:
+    extends:
+      file: ../common-services.yaml
+      service: coreapi
+    volumes:
+      - ../alive/keys:/.keys:ro
+      - ../alive/appsettings.json:/app/API/appsettings.json:ro
+    environment:
+      LEAF_JWT_KEY_PW: ${alive_JWT_KEY_PW}
```

Could you switch the site name to uppercase in variable names? 

No strong preference either way, just want consistency so they're easy to use in scripts eg [this snippet](https://github.com/uwcirg/leaf-environments?tab=readme-ov-file#generate-a-signing-key)

```suggestion
      LEAF_JWT_KEY_PW: ${ALIVE_JWT_KEY_PW}
```

---

Could you add this (and the other) site-specific JWT password variables to `default.env` as a "stub", for reference? 

Having all the required config variables in one place makes it easier to configure `.env` in one pass